lua/ts-node-action/init.lua

```diff
@@ -19,11 +19,11 @@ local function replace_node(node, replacement, opts)
     start_row, start_col, end_row, end_col, replacement
   )
 
-  if opts.strip_whitespace then
```

Comment:
Could the same thing be accomplished by passing a callback function?

Reply:
I'll answer my own question: yes, but, I also don't think it's unreasonable to expose this as an option, as it's pretty similar to how we do formatting.